modules/reindex/src/main/java/org/elasticsearch/index/reindex/RestUpdateByQueryAction.java

```diff
@@ -127,6 +128,15 @@ static Script parseScript(Map<String, Object> config, ParseFieldMatcher parseFie
                     .getPreferredName(), ScriptType.STORED.getParseField().getPreferredName());
         }
         assert type != null : "if script is not null, type should definitely not be null";
-        return new Script(script, type, lang, params);
+
+        if (lang == null) {
+            lang = DEFAULT_SCRIPT_LANG;
```

Comment:
Could use `Collections.emptyMap()`? If not then I agree, keeping it down here is appropriate.